.serena/memories/project_overview.md

```
@@ -28,6 +28,10 @@ This is a **cross-platform Nix configuration** for macOS (nix-darwin) and NixOS
 - `wrappers` (nix-wrapper-modules) - Neovim plugin/LSP management via Nix
 - `vim-tidal-lua` - TidalCycles Neovim plugin (Lua rewrite)
 
+## Configuration Pattern
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)